omod/src/main/webapp/pages/htmlform/IllegalArgumentException.gsp

```diff
@@ -0,0 +1,12 @@
+<html>
+   <body>
+   <p>Application has encountered an error. Please contact support on ...</p>
+     <% out << "encounter.form is not an HTML Form" %>
```

Comment:
is this the error you are returning? down you controller ,add the error message to an attribute for example errorMessage and access it from the gsp as ${errorMessage }

Reply:
thanks @HerbertYiga , ,Am not well convinced with this approach and am sure there is a clean way of doing this which am looking forward too.